src/components/ha-faded.ts

```diff
@@ -0,0 +1,70 @@
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, query, state } from "lit/decorators";
+import { classMap } from "lit/directives/class-map";
+
+@customElement("ha-faded")
+class HaFaded extends LitElement {
+  @state() faded = true;
+
+  @state() fadeable = true;
+
+  @query(".container") private _container!: HTMLDivElement;
+
+  protected render(): TemplateResult {
+    return html`
+      <div
+        class="container ${classMap({
+          fadeable: this.fadeable,
+          faded: this.fadeable && this.faded,
+        })}"
+        @click=${this._toggleContainer}
+      >
+        <slot></slot>
+      </div>
+    `;
+  }
+
+  protected firstUpdated(changedProps) {
+    super.firstUpdated(changedProps);
+    if (
+      this._container.clientHeight !== 0 &&
+      this._container.clientHeight < 112
```

Comment:
Maybe for another PR. But should we make this configurable? probably not until we need it I guess